cmd/kail/main.go

```diff
@@ -72,6 +72,9 @@ var (
 	flagGlogVmodule = kingpin.Flag("glog-vmodule", "glog -vmodule flag").
 			Default("").
 			String()
+	flagJsonPP = kingpin.Flag("jpp", "Json Pretty Print. If log lines are JSON, then pretty print them.").
```

Comment:
I'd prefer `--json-pp` or even `--writer=json-pp` (implement via a different writer instead of modifying current one)

Reply:
Done.